src/etc/inc/notices.inc

```diff
@@ -410,10 +410,60 @@ function send_smtp_message($message, $subject = "(no subject)", $force = false)
 	log_error(sprintf(gettext("Message sent to %s OK"), $to));
 	return;
 }
+/****f* notices/notify_via_telegram
+ * NAME
+ *   notify_via_telegram
+ * INPUTS
+ *	 notification string to send to Telegram via API
+ * RESULT
+ *   returns NULL if message was sent
+ ******/
+
+function notify_via_telegram($message, $force = false) {
+	global $config;
+	if ((!isset($config['notifications']['telegram']['enabled']) || (!$force))
+	|| (!$config['notifications']['telegram']['api'] || !$config['notifications']['telegram']['chatid'])) {
+		if($force) {
+			$err_msg="Unable to test Telegram notification without both API Key & Chat ID set";
+			return $err_msg;
```

Comment:
If you are `return`-ing the value right after setting it, why bother setting it? Just return the string.

Also the string should be wrapped in `gettext()`.